tests/src/ViewModel/InfoBarTest.php

```diff
@@ -37,17 +50,18 @@ public function it_cannot_have_blank_text()
     /**
      * @test
      */
-    public function it_cannot_have_an_invalid_type()
```

Comment:
the original test should not disappear, duplicate rather than change so that both `it_cannot_have_an_invalid_type` and `dismissible_type_must_have_id` are present

Reply:
Removed in error. Reinstated in https://github.com/elifesciences/patterns-php/pull/706/commits/dc9312afdb581c645c235cb50dd52a40eedbaf49 (the diff in show slightly oddly right now, you'll see it's back on [line 43, above](https://github.com/elifesciences/patterns-php/pull/706/files#diff-ea0577fdd66a883566c7f522a10bb0bcR43)).